### DISCUSSION TITLE
blog/remain-conscious/

### DISCUSSION THREAD

**Discussion by giscus** [152334H/152334H.github.io#13] [Announcements]
# blog/remain-conscious/

I don’t think very often. These are some of the things I’ve tried in the past to rectify that.

https://152334h.github.io/blog/remain-conscious/

**Comment by filipviz** [comment]
I think of this often.

**Reply by 152334H** [reply]
adding: meditation works. speaking has less retention than writing. ambient temperature matters but not sure how much. songs are training wheels for the unfocused and your best thoughts will move faster in silence.

but most of all: the specific tips don't really matter much, finding innate joy in consciousness does. good agents will seek rewards to the ends of the earth
